packages/react-components/react-dialog/src/components/DialogTrigger/useDialogTrigger.ts

```diff
@@ -39,6 +39,7 @@ export const useDialogTrigger_unstable = (props: DialogTriggerProps): DialogTrig
     children: applyTriggerPropsToChildren<DialogTriggerChildProps>(
       children,
       useARIAButtonProps(child?.type === 'button' || child?.type === 'a' ? child.type : 'div', {
+        type: 'button',
```

Comment:
shouldn't `useARIAButton` set the type by default ?

Reply:
That's a valid point, I'll open an issue to update `useARIAButtonProps` on that.